spec/factories/reviews.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :review do
+    reviewer 'rev1'
+    rating 3
+    comment 'comment long so very long comment 1'
+    restaurant { create(:restaurant) }
+  end
+
+  trait :invalid_rating do
```

Comment:
Not sure I love that uses of traits

Reply:
same here, though it's an interesting idea. usually we only use traits that are valid.